crowbar_framework/vendor/assets/javascripts/jquery/ledUpdate.js

```diff
@@ -242,6 +235,9 @@
   };
 
   LedUpdate.prototype.update = function(element, clazz, title, callback) {
+    var self = this;
+    var animatedClasses = ['unready', 'in_process', 'spin'];
```

Comment:
@cyntss I guess we can drop this 2 variables? looks like a remainder from debugging